.github/workflows/test.yaml

```
@@ -5,6 +5,7 @@ permissions:
 
 on:
   push:
+    branches: ['main', 'rel-*', 'ci/*']
   pull_request:
```

```suggestion
  pull_request:
  workflow_dispatch:
```

This would mean, in the case where some maintainer really does want to run CI for something without opening a PR, they can do that by pushing the branch and then running it manually (this enables the UI for that).